### ISSUE TITLE
linter: `just new-jsx-a11y-rule` command fails

### ISSUE THREAD

**Issue by mehm8128** [oxc-project/oxc#21768]
### What version of Oxlint are you using?

latest

### What happened?

When running `just new-jsx-a11y-rule some-rule-name`, it fails.

```
just new-jsx-a11y-rule no-interactive-element-to-noninteractive-role
    Finished `dev` profile [unoptimized] target(s) in 7.89s
     Running `target/debug/rulegen no-interactive-element-to-noninteractive-role jsx-a11y`
Reading test file from https://raw.githubusercontent.com/jsx-eslint/eslint-plugin-jsx-a11y/main/__tests__/src/rules/no-interactive-element-to-noninteractive-role-test.js
File parsed and 3 pass cases, 2 fail cases are found
Reading rule source file from https://raw.githubusercontent.com/jsx-eslint/eslint-plugin-jsx-a11y/main/src/rules/no-interactive-element-to-noninteractive-role.js

thread 'main' (261810) panicked at tasks/rulegen/src/main.rs:1581:13:
assertion failed: ret.errors.is_empty()
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: Recipe `new-rule` failed on line 173 with exit code 101
```

The root cause appears to be a Flow parsing error.
`eslint-plugin-jsx-a11y` uses Flow ([ref](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/8f75961d965e47afb88854d324bd32fafde7acfe/package.json#L83)), so the parser in`tasks/rulegen/src/main.rs` can't handle the source file, causing the assertion to fail.
Commenting out `assert!(ret.errors.is_empty())` allows the rule template to be generated successfully, so this regression was likely introduced by [test(all): check parse errors existence by sapphi-red · Pull Request #15865 · oxc-project/oxc](https://github.com/oxc-project/oxc/pull/15865).
